perf_test/blas/blas2/tracked_testing.hpp

```diff
@@ -0,0 +1,65 @@
+//
+// Created by Poliakoff, David Zoeller on 4/26/21.
+//
+#ifndef KOKKOSKERNELS_BLAS2_TRACKED_TESTING_HPP
+#define KOKKOSKERNELS_BLAS2_TRACKED_TESTING_HPP
+
+#include <common/RAJAPerfSuite.hpp>
+#include <common/Executor.hpp>
+
+#include "KokkosBlas2_gemv_perf_test.hpp"
+
+namespace test {
+namespace blas2 {
+
+// Register kernels per test
+
+void build_gemv_executor(rajaperf::Executor& exec,
+                         int argc,
+                         char* argv[],
+                         const rajaperf::RunParams& params) {
+        for(auto* kernel : construct_gemv_kernel_base(params)) {
+                exec.registerKernel("BLAS2", kernel);
+        }
+}
+
+
+/*
```

Comment:
Clean up commented out code

Reply:
@brian-kelley -- done!  I'll push updates to this PR, and make a final check with you to make sure it's OK to merge into develop.  Thank you for all your help!